default.yml

```diff
@@ -32,8 +36,20 @@ Metrics/CyclomaticComplexity:
 
 Metrics/LineLength:
   Max: 120
+  Exclude:
+    - config/routes.rb
+    - config/routes/*
+    - db/migrate/*
+    - spec/**/*
```

Comment:
Thanks for doing this, @jlw! It sounds like your team has started really blazing the trail with RamseyCop. 

Removing the line length requirement from tests and specs is the only think I've got a concern with.
Do you really need more than 120 characters? Is this because of nested contexts?

Reply:
I often run afoul of this with hard-coded strings in expectations.

Reply:
Interesting. That is one long string. I'm not opposed, just trying to understand how big of a deal it really is.

Reply:
Looking through my tests, it was actually an array of data, not a string, though I am pretty sure I have seen that too somewhere.

Reply:
I agree with @breese524 because most of the line length offenses I've had to correct in my tests did not come from super-long strings, but I have had a few super-long strings. I don't see a reason to exclude exclude every part of the spec files from this rule.

Reply:
but the rest of these are a welcome addition since I've had to add them to my .rubocop.yml myself haha

Reply:
Our FPU test line length offenses are a mixture of nested enumerables, long comparison strings, and adding custom failure messages - sometimes together (and nested contexts add a little). I would argue more for line length exclusions in the other locations and not so much on excluding tests.

Reply:
(I've reverted that last commit for review with the intent to simply pop those commits off the branch before merging.)